app/src/components/nav-menu/UserHomePage.js

```diff
@@ -0,0 +1,223 @@
+import React, {Component} from 'react';
+import {Redirect} from 'react-router-dom';
+import UsersDataRequests from '../../common/UsersDataRequests'
+import endPointsList from '../../common/endPointsList';
+import NavigateMenu from './NavigateMenu'
+import {Validations} from '../../common/Validations';
+
+import '../../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import '../../css/App.css';
+import {connect} from "react-redux";
+
+class UserHomePage extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            isEdit: false,
+            isLoggedIn: true,
+            isNeedUserInfo: true,
+            userInfo: {},
+            formattedUserInfo: ''
+        };
+
+        this.loadUserInfo();
+
+        this.onSignOut = this.onSignOut.bind(this);
+        this.onInfoClick = this.onInfoClick.bind(this);
+        this.handleSaveInfoClick = this.handleSaveInfoClick.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(event){
+        const thisInput = {
+            content: '',
+            // isValid: false
+        };
+        // thisInput.isValid = Validations.typeOfRegexp(event.target.value, typeOfRegexp);
+        thisInput.content = event.target.value;
+
+        this.props.changeUserInfoFormInput({key: event.target.name, value: thisInput});
+    }
+
+    onSignOut() {
+        localStorage.removeItem('auth-tok');
+        this.setState({...this.state, isLoggedIn: false});
+    }
+
+    isTokenGood() {
```

Comment:
I believe you should move this check to common function. Also, would the token be still 'good' if I put to the localStorage in auth-tok field '123blablabla321' ?